src/moore/static/sass/partials/blocks/_two-column-grid.scss

```
@@ -0,0 +1,45 @@
+
+.block-two_column_grid {
+  padding-bottom: 120px;
+  padding-top: 120px;
+  
+  .row-reverse {
+    flex-direction: row-reverse;
+  }
+  
+  .two-column-row {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    
+    @media #{$medium-and-down} {
+      flex-direction: column;
+      align-items: stretch;
+    }
+  }
+  
+  .row-paragraph {
+    flex: 1 0 50%;
+    overflow-y: scroll;
```

Använd overflow-y: auto istället. I chrome får man en scrollbar även om texten är kort nog att man inte behöver en scrollbar. Jag länkar en bild i koversationen
```suggestion
    overflow-y: auto;
```